v3/mqtt_mode.go

```diff
@@ -13,6 +13,11 @@ import (
 	"time"
 )
 
+const (
+	DefaultUseTLS    = false
```

Comment:
IMO the default should be true. Because we highly recommend to use TLS.
https://dev.tinkermode.com/doc/platform/how-to/mqtt-with-mode

Reply:
Sounds good to me.